### PR TITLE
Feat/upload images

### PR DESCRIPTION
**Pull request by BishwasKoirala**
made routes for uploading and deleting images.
with swagger docs

### CODE REVIEW

File: storage/api-docs/api-docs.yaml [hunk 1]
```diff
@@ -316,3 +353,24 @@ components:
...
+      properties:
+        image_id:
+          type: string
+          example : 1yU3dbWNL2Hr-FSOATrFCbJBGakGagFPd
+        image_name:
+          type: string
+          example: 20241031-VW26J3
+        image_status:
+          type: string
+          example: uploaded | deleted
```
Comment (rajanvalencia) [inline storage/api-docs/api-docs.yaml:372]:
```suggestion
        image_status:
          type: string
          enum:
            - uploaded
            - deleted
          example: "uploaded"
```

File: routes/api.php [hunk 2]
```diff
@@ -2,17 +2,23 @@
...
 
-Route::prefix('webhooks')->group(function () {
+Route::prefix('/webhooks')->group(function () {
     Route::post('/payment-intent-succeed', [WebhookController::class, 'paymentIntentSucceed']);
 });
 Route::post('/webhook', [WebhookController::class, 'create']);
 
+Route::prefix('/images')->group(function () {
+    Route::post('/upload-donation-image', [DonationImageController::class, 'uploadDonationImage']);
+    Route::delete('/delete-donation-image/{image_id}', [DonationImageController::class, 'deleteDonationImage']);
```
Comment (rajanvalencia) [inline routes/api.php:19]:
```suggestion
    Route::post('/upload', [DonationImageController::class, 'uploadDonationImage']);
    Route::delete('/delete/{image_id}', [DonationImageController::class, 'deleteDonationImage']);
```

File: storage/api-docs/api-docs.yaml [hunk 3]
```diff
@@ -28,6 +28,42 @@ paths:
...
+                  type: string
+                  format: binary
+      responses:
+        200:
+          description: Success
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Image'
+  /api/images/delete-donation-image/{image_id}:
```
Comment (rajanvalencia) [inline storage/api-docs/api-docs.yaml:54]:
```suggestion
  /api/images/delete/{image_id}:
```

File: storage/api-docs/api-docs.yaml [hunk 4]
```diff
@@ -28,6 +28,42 @@ paths:
               schema:
                 $ref: "#/components/schemas/DonationResponse"
 
+
+  /api/images/upload-donation-image:
```
Comment (rajanvalencia) [inline storage/api-docs/api-docs.yaml:32]:
```suggestion
  /api/images/upload:
```

**Review by rajanvalencia — CHANGES_REQUESTED** [review]

**Review by rajanvalencia — APPROVED** [review]
LGTM👍

**Comment by BishwasKoirala** [comment]
LGTM

**Pull request merged**